Pipfile

```diff
@@ -20,6 +20,7 @@ uvicorn = "==0.14.0"
 nkeys = "==0.1.0"
 linuxforhealth-edi = "==0.28.0"
 python-multipart = "==0.0.5"
+charset-normalizer = "==2.0.7"
```

Comment:
Fixing charset-normalizer at 2.0.7 avoids a warning message on every POST to LFH which occurs with the 2.0.8 version